tests/test_cli_options.py

```diff
@@ -0,0 +1,507 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from distutils.cmd import Command
+from email.policy import default
+import os
+import copy
+
+from model_analyzer.config.input.config_defaults import DEFAULT_TRITON_DOCKER_IMAGE
+
+from .common import test_result_collector as trc
+
+from model_analyzer.cli.cli import CLI
+from model_analyzer.config.input.config_command_profile import ConfigCommandProfile
+from model_analyzer.config.input.config_command_analyze import ConfigCommandAnalyze
+from model_analyzer.config.input.config_command_report import ConfigCommandReport
+from model_analyzer.config.input.config_status import ConfigStatus
+from model_analyzer.constants import CONFIG_PARSER_SUCCESS
+
+import psutil
+
+from unittest.mock import patch
+
+
+class CLISubclass(CLI):
```

Comment:
I feel like this could have been done with a single patch or some other way of faking out the ARGV.  
This maybe? @matthewkotila 
https://github.com/triton-inference-server/model_analyzer/blob/main/tests/test_cli.py#L35

Also, should that file be combined into this one?